examples/guided.py

```diff
@@ -150,48 +186,76 @@ def ask_user_questions():
 	if not archinstall.arguments.get('hostname', None):
-		archinstall.arguments['hostname'] = input('Desired hostname for the installation: ').strip(' ')
+		archinstall.arguments['hostname'] = input(
+			'Desired hostname for the installation: '
+		).strip(' ')
 
 	# Ask for a root password (optional, but triggers requirement for super-user if skipped)
 	if not archinstall.arguments.get('!root-password', None):
-		archinstall.arguments['!root-password'] = archinstall.get_password(prompt='Enter root password (Recommendation: leave blank to leave root disabled): ')
+		archinstall.arguments['!root-password'] = archinstall.get_password(
+			prompt='Enter root password (Recommendation: leave blank to leave root disabled): '
+		)
 
 	# Ask for additional users (super-user if root pw was not set)
 	archinstall.arguments['users'] = {}
 	archinstall.arguments['superusers'] = {}
 	if not archinstall.arguments.get('!root-password', None):
-		archinstall.arguments['superusers'] = archinstall.ask_for_superuser_account('Create a required super-user with sudo privileges: ', forced=True)
+		archinstall.arguments['superusers'] = archinstall.ask_for_superuser_account(
+			'Create a required super-user with sudo privileges: ', forced=True
+		)
 
-	users, superusers = archinstall.ask_for_additional_users('Enter a username to create a additional user (leave blank to skip & continue): ')
+	users, superusers = archinstall.ask_for_additional_users(
+		'Enter a username to create a additional user (leave blank to skip & continue): '
+	)
 	archinstall.arguments['users'] = users
-	archinstall.arguments['superusers'] = {**archinstall.arguments['superusers'], **superusers}
+	archinstall.arguments['superusers'] = {
+		**archinstall.arguments['superusers'],
+		**superusers,
+	}
 
 	# Ask for archinstall-specific profiles (such as desktop environments etc)
 	if not archinstall.arguments.get('profile', None):
-		archinstall.arguments['profile'] = archinstall.select_profile(archinstall.list_profiles(filter_top_level_profiles=True))
+		archinstall.arguments['profile'] = archinstall.select_profile(
+			archinstall.list_profiles(filter_top_level_profiles=True)
+		)
 	else:
-		archinstall.arguments['profile'] = archinstall.list_profiles()[archinstall.arguments['profile']]
+		archinstall.arguments['profile'] = archinstall.list_profiles()[
+			archinstall.arguments['profile']
+		]
 
 	# Check the potentially selected profiles preparations to get early checks if some additional questions are needed.
-	if archinstall.arguments['profile'] and archinstall.arguments['profile'].has_prep_function():
-		with archinstall.arguments['profile'].load_instructions(namespace=f"{archinstall.arguments['profile'].namespace}.py") as imported:
+	if (
+		archinstall.arguments['profile']
+		and archinstall.arguments['profile'].has_prep_function()
+	):
+		with archinstall.arguments['profile'].load_instructions(
+			namespace=f"{archinstall.arguments['profile'].namespace}.py"
+		) as imported:
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.